libs/core/langchain_core/runnables/fallbacks.py

```diff
@@ -172,9 +174,12 @@ def invoke(
             try:
                 if self.exception_key and last_error is not None:
                     input[self.exception_key] = last_error
-                output = runnable.invoke(
+                child_config = patch_config(config, callbacks=run_manager.get_child())
```

Comment:
Do we need similar changes in RunnableParallel? 

Is there any utility code that can be pulled out?

Reply:
Likely ya

Reply:
Maybe `patch_and_set_context()` or something

Reply:
Decided against for now